### ISSUE TITLE
Implement accessibility and supportability

### ISSUE THREAD

**Issue by Hkhan221** [Hkhan221/3D-WorkShop#17]
https://www.w3.org/standards/webdesign/accessibility

Images should include equivalent alternative text (alt text) in the markup/code.

Images should include equivalent alternative text (alt text) in the markup/code.

If alt text isn't provided for images, the image information is inaccessible, for example, to people who cannot see and use a screen reader that reads aloud the information on a page, including the alt text for the visual image.

When equivalent alt text is provided, the information is available to people who are blind, as well as to people who turn off images (for example, in areas with expensive or low bandwidth). It's also available to technologies that cannot see images, such as search engines.

Keyboard Input
mouse crossed out
Some people cannot use a mouse, including many older users with limited fine motor control. An accessible website does not rely on the mouse; it makes all functionality available from a keyboard. Then people with disabilities can use assistive technologies that mimic the keyboard, such as speech input.
